src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -301,7 +305,7 @@ export const ConsultationDetails = (props: any) => {
                       className={tabButtonClasses(tab === p)}
```

Comment:
Read the list of tabs from constants file

Reply:
done